include/ttmlir/Dialect/TTIR/IR/TTIROps.td

```diff
@@ -340,6 +340,62 @@ def TTIR_TransposeOp : TTIR_DPSOp<"transpose"> {
     let hasVerifier = 1;
 }
 
+def TTIR_Conv2dOp : TTIR_DPSOp<"conv2d"> {
+    let summary = "Conv2d operation.";
+    let description = [{
+      Conv2d operation.
```

Comment:
This is more for MLIR owners to confirm, but it'll be good to focus on more detailed descriptions of the ops we're bringing up.